### PR TITLE
oVirt: create workflows for specific releases

### PR DESCRIPTION
**Pull request by Gal-Zaidman**
On oVirt we have minor changes between the releases logic that require different logic.

This PR will solve the immediate problem of disabling the Affinity Groups feature on 4.8 and above runs, we need to disable it because it can cause infra-related failures in case the Affinity Groups constrains can't be fulfilled and we want to avoid that.

Since no we needed 3 different types of workflows which all have minor differences it is easier and clearer to pin the workflow name to the release.
 
4.4,4.5 releases:
- Needs to be configured without CSI and Monitoring PVC.
- Use insecure connection to oVirt on the installer.
- Use dns_vip on install-config.

4.6,4.7 releases:
- Use insecure connection to oVirt on the installer.

4.8 and above:
- Need to disable the Affinity Groups feature on the install config
- Use secure connection to oVirt on the installer.

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.6-4.7/openshift-e2e-ovirt-conformance-release-4.6-4.7-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    test:
+    - ref: openshift-e2e-ovirt-minimal-test
+    post:
+    - chain: ipi-ovirt-post
+    env:
+      TEST_TYPE:  "suite"
+      TEST_SUITE: "openshift/conformance/parallel"
+      IGNITIONVERSION: "2.2.0"
+  documentation: |-
+    The Openshift E2E oVirt e2e without csi workflow installs OpenShift cluster on top of oVirt
```
Comment (eslutsky) [inline ci-operator/step-registry/openshift/e2e/ovirt/conformance/release-4.6-4.7/openshift-e2e-ovirt-conformance-release-4.6-4.7-workflow.yaml:16]: "e2e without csi" - is that the old name ?

File: ci-operator/step-registry/ipi/conf/ovirt/release-4.4-4.5/ipi-conf-ovirt-release-4.4-4.5-chain.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+chain:
+  as: ipi-conf-ovirt-release-4.4-4.5
+  steps:
+  - ref: ipi-conf-ovirt-generate-install-config-params
+  - ref: ipi-conf-ovirt-generate-ovirt-config-insecure
+  - ref: ipi-conf-ovirt-generate-install-config-release-4.4-4.5
+  documentation: |-
+    The IPI configure step chain generates the install-config.yaml and ovirt-config.yaml files
+    based on the cluster profile and lease inforamtion for a release-4.4/4.5 OpenShift cluster.
```
Comment (eslutsky) [inline ci-operator/step-registry/ipi/conf/ovirt/release-4.4-4.5/ipi-conf-ovirt-release-4.4-4.5-chain.yaml:9]: inforamtion = information

**Review by eslutsky — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/ipi/conf/ovirt/release-4.4-4.5/ipi-conf-ovirt-release-4.4-4.5-chain.yaml (hunk 2 above)
Comment (Gal-Zaidman) [reply ci-operator/step-registry/ipi/conf/ovirt/release-4.4-4.5/ipi-conf-ovirt-release-4.4-4.5-chain.yaml:9]: Done

File: ci-operator/step-registry/ipi/ovirt/pre/upgrade/ipi-ovirt-pre-upgrade-chain.yaml [hunk 3]
```diff
@@ -6,4 +6,4 @@ chain:
   - chain: ipi-install-ovirt-stableinitial
   documentation: |-
     The IPI setup step contains all steps that provision an OpenShift cluster
-    with persistent Prometheus storage.
+    with persistent Prometheus storage and no Affiniry Groups.
```
[the remark this answers is not in the document]
Comment (Gal-Zaidman) [reply ci-operator/step-registry/ipi/ovirt/pre/upgrade/ipi-ovirt-pre-upgrade-chain.yaml:9]: Done

File: ci-operator/step-registry/openshift/e2e/ovirt/minimal/release-4.6-4.7/openshift-e2e-ovirt-minimal-release-4.6-4.7-workflow.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    post:
+    - chain: ipi-ovirt-post
+    env:
+      TEST_SKIPS: "Early"
+      TEST_TYPE:  "suite"
+      IGNITIONVERSION: "2.2.0"
+  documentation: |-
+    The Openshift E2E oVirt e2e minimal release 4.6,4.7 workflow installs OpenShift cluster on top of oVirt
+    from releases that didn't contain affinity groups and secure connection to the oVirt engine from the installer,
+    therfor it uses a different install config and ovirt-config.
```
[the remark this answers is not in the document]
Comment (Gal-Zaidman) [reply ci-operator/step-registry/openshift/e2e/ovirt/minimal/release-4.6-4.7/openshift-e2e-ovirt-minimal-release-4.6-4.7-workflow.yaml:18]: Done

File: ci-operator/step-registry/openshift/e2e/ovirt/csi/release-4.6-4.7/openshift-e2e-ovirt-csi-release-4.6-4.7-workflow.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    post:
+    - chain: ipi-ovirt-post
+    env:
+      TEST_TYPE:  "suite"
+      TEST_SUITE: "openshift/csi"
+      TEST_CSI_DRIVER_MANIFEST: csi-test-manifest.yaml
+  documentation: |-
+    The Openshift E2E oVirt `csi` release 4.6,4.7 workflow installs OpenShift cluster on top of oVirt
+    from releases that didn't contain affinity groups and secure connection to the oVirt engine from the installer,
+    therfor it uses a different install config and ovirt-config.
```
[the remark this answers is not in the document]
Comment (Gal-Zaidman) [reply ci-operator/step-registry/openshift/e2e/ovirt/csi/release-4.6-4.7/openshift-e2e-ovirt-csi-release-4.6-4.7-workflow.yaml:17]: Done

**Comment by jsafrane** [comment]
/approve
the storage parts

**Comment by sinnykumari** [comment]
/lgtm
For MCO part

**Pull request merged**